zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/JdbcInterpreterResult.java

```diff
@@ -0,0 +1,140 @@
+package org.apache.zeppelin.interpreter;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+
+import javax.sql.rowset.RowSetProvider;
+import javax.sql.rowset.CachedRowSet;
+import java.sql.SQLException;
+
+
+import org.apache.commons.codec.binary.StringUtils;
+import java.util.UUID;
+import com.google.gson.GsonBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Unlike a regular interpreter result, a JdbcInterpreter Result caches its 
+ * output so that it can be used later and persisted independently of the regular result.
+ * It also has a standard return for all tabular SQL data. 
+ * @author Rusty Phillips {@literal: <rusty@cloudspace.com>}
```

Comment:
Would you be willing to remove `@author` tag from JavaDoc please?
Although not clearly documented we, as many other ASF projects, do not use them to keep track of contributions, but use git logs and JIRA issues instead.